src/app/components/VerifyingIfAnswerIsCorrect.js

```diff
@@ -0,0 +1,11 @@
+
+export default function VerifyingAnswerIsCorrect() {
```

Comment:
I think you should export only one function. Right now if you import VerifyingAnswerIsCorrect(), there is no way to call verifyingAnswer.

Reply:
I changed function, now is only one.